frontend/src/views/SignUpView.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div>
+    <h1>S'Inscrire</h1>
+    <form @submit.prevent="signup" class="mx-auto p-3">
+      <div class="bg-danger text-white p-1 mx-4 rounded" v-if="erreurs.length">
+        <span v-for="erreur in erreurs" :key="erreur">{{ erreur }}</span>
+      </div>
+      <input
+        class="text-center p-1 my-3 rounded"
+        v-model="courriel"
+        placeholder="Courriel"
+      />
+      <div class="w-100"></div>
+      <input class="text-center p-1 rounded" v-model="nom" placeholder="Nom" />
+      <div class="w-100"></div>
+      <input
+        class="text-center p-1 my-3 rounded"
+        v-model="password"
+        placeholder="Mot de passe"
+        type="password"
+      />
+      <div class="w-100"></div>
+      <button class="btn btn-success px-5" type="submit">Créer</button>
+      <div class="my-3" style="width: 100%"></div>
+      <router-link
+        to="/auth/login"
```

Comment:
This need to be "/login" because this is the frontend route to access login and not the backend endpoint 😉

Reply:
Alright got it